bin/utils/subkey/src/main.rs

```diff
@@ -329,13 +396,24 @@ where
 	if let Some(network) = maybe_network {
 		set_default_ss58_version(network);
 	}
+
+	let output: OutputType = match matches.value_of("output").map(|output| {
+		output
+			.try_into()
+			.map_err(|_| Error::Static("Invalid output name. See --help for available outputs."))
```

Comment:
You should move this error down to the line `405`.

Than you can change the map to `map(TryInto::tryinto)`